data/wanderers middle.txt

```diff
@@ -1766,7 +1766,73 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+			choice
+				`	"You're lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm looking for."`
+					goto thanks
+					
+			label where
+			`	"Well," the pilot explains, "we were doing a large cargo run to Farpoint in the Alnitak system, when two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies in this area. I had to jump the ship in atmosphere to stop us from getting shredded by their strange melting weaponry. I immediately reported the attack to a desk clerk at the Navy base in the neighboring system Saiph, but their servers show no logs of any unidentified ships beyond Farpoint, so I guess they turned around after scouting out the Alnitak system."`
+			choice
+				`	"You're lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm hunting for."`
+					goto thanks
+			
+			label thanks
+			`	"No problem, kiddo," the captain says, although he looks awed that you are even attempting to fight such powerful ships. "Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in one of those pirate systems up there. Take care of yourself out there; those are some dangerous ships that jumped us." With that, he bids you farewell.`
+					decline
+					
+			label glad
+			`	"Yeah, those ships would have obliterated us if we hadn't jumped pronto." The pilot looks at you, then asks quizzically, "Say, do you know anything about those ships?"`
+			choice
+				`	"Yeah, those were Korath ships that attacked you."`
+				`	"Maybe. I have a feeling that they're the alien ships that I'm hunting."`
+				`	"I don't want to jump to any conclusions."`
+					goto no
+					
+			`	The pilot seems to display an expression of distaste upon hearing of aliens. You try to describe the dynamic between the exiles, the Efreti, Mereti, and Sestor, but to no avail whatsoever. The pilot quickly loses interest in your explanation, so you decide to steer the conversation to an end.`
+				goto end
+				
+			label no
+			`	The pilot seems to be disappointed in your response, as if he wanted to learn more of the mysterious ships that attacked the Heart of Gold, so you decide to steer the conversation to an end.`
+				goto end
+				
+			label end
+			`	"Thank you for your time," you say to the captain. The pilot quickly perks up again, and says, "No problem, kiddo. Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in one of those pirate systems up there. Take care of yourself out there; if they really are alien ships, they must be pretty dangerous." With that, he bids you farewell.`
```

Comment:
It's almost a valid point, but as @Amazinite points out, it's used in the in-game text already. And also,  If James calls you a "kid", would you find him to be a condescending jerk?

If your answer was yes, well, he _does_ do so. "There's no quick road to riches, kid."  The Behemoth captain is meant to be an old-timer who is experienced at the game, as demonstrated by his witty decision to jump instead of land. Almost like a James that hasn't quite yet reached retirement. I personally don't think it's condescending, and didn't intend it to be so.